brave/brave-core
# Import source profiles to separate profiles

Resolves https://github.com/brave/brave-browser/issues/27733
Resolves https://github.com/brave/brave-browser/issues/27591

Removed parallel import feature because the onboarding has been merged.

Chrome:

https://user-images.githubusercontent.com/2965009/212100704-677386bd-3b69-479f-8d09-2b5183643906.mp4

Firefox:

https://user-images.githubusercontent.com/2965009/212103552-797d7f75-7015-423b-b53d-7fb8c80bde30.mp4


## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Import multiple profiles from different browsers
- Check each profile imported to separate Brave's profile
- Avatars for newly created profiles should be assigned randomly
- If user importing only one source profile it should not create a new Brave profile and import to Default one
- Steps:
  - Install Chrome/Firefox/any other and create there 3-4 profile with own bookmarks, history, extensions, password etc.
  - Go to brave welcome page and go through onboarding flow
  - All profiles should be created in Brave with same names and contain imported data
